.gitignore

```diff
@@ -1,168 +1,22 @@
-# Ignore created packages
-/package
-
-/.idea
-
-# Byte-compiled / optimized / DLL files
-__pycache__/
-*.py[cod]
-*$py.class
```

Comment:
Why?

Reply:
This is python default
```
# Byte-compiled / optimized / DLL files
__pycache__/
*.py[cod]
*$py.class
```

Reply:
I have reverted that deletion. 
https://github.com/ynput/example-studio-addon/pull/3/commits/7c00197b79cfc43dbee52e9922f0aa486ec2a6bd

Reply:
Aah, github not showing me 😥